clients/callback_client.go

```diff
@@ -46,8 +46,10 @@ func NewPeriodicCallbackClient(callbackInterval time.Duration) *PeriodicCallback
 func (pcc *PeriodicCallbackClient) Start() *PeriodicCallbackClient {
 	go func() {
 		for {
-			recoverer(pcc.SendCallbacks)
-			time.Sleep(pcc.callbackInterval)
+			recoverer(func() {
+				pcc.SendCallbacks()
+				time.Sleep(pcc.callbackInterval)
```

Comment:
Maybe sleep before calling `SendCallbacks` to make sure to still backoff if `SendCallbacks` is panicking

Reply:
I thought that too, but then you always delay the first update. I think that would be fine in reality, but we might need to go and update some unit tests.

Reply:
Decided to stop being lazy and just fix this now